includes/Classifai/Providers/Azure/ComputerVision.php

```diff
@@ -199,7 +199,7 @@ public function sanitize_settings( $settings ) {
 			);
 		}
 
-		if ( is_numeric( $settings['caption_threshold'] ) && (int) $settings['caption_threshold'] >= 0 && (int) $settings['caption_threshold'] <= 100 ) {
+		if ( isset( $settings['caption_threshold'] ) && is_numeric( $settings['caption_threshold'] ) && (int) $settings['caption_threshold'] >= 0 && (int) $settings['caption_threshold'] <= 100 ) {
```

Comment:
Not having this check was causing some tests to error.

Reply:
nice catch!